### ISSUE TITLE
header (h3) in the form element should have an accessible name

### ISSUE THREAD

**Issue by greenervigil** [cengage/react-magma#660]
**Describe the bug**
The H3 header in the form component is missing an accessibility aria-labelledby.

**Expected behavior**
H3 in the Form component should be read by a screen reader.

**Additional context**
Reported through Cengage Jira project with ticket  https://jira.cengage.com/browse/B2CFE-2

**Comment by greenervigil** [comment]
Closing this ticket.  This is a misuse of the form element and not an actual bug.
